services/core-web/cypress/e2e/majorprojects.cy.ts

```diff
@@ -1,100 +1,98 @@
 describe("Major Projects", () => {
-    beforeEach(() => {
-        cy.login();
+  beforeEach(() => {
+    cy.login();
 
-        cy.get('[data-cy="home-link-button-major-projects"]', { timeout: 5000 }).click({ force: true });
+    cy.get('[data-cy="home-link-button-major-projects"]', { timeout: 5000 }).click({ force: true });
 
-        // .eq(1) selects the second row (0-based index).
-        cy.get("[data-cy=major-projects-table-open-button]", { timeout: 5000 })
-            .eq(1)
-            .find("button")
-            .click({ force: true });
+    // .eq(1) selects the second row (0-based index).
+    cy.get("[data-cy=major-projects-table-open-button]", { timeout: 5000 })
+      .eq(1)
+      .find("button")
+      .click({ force: true });
 
+    cy.get('a[data-cy="project-description-view-link"]', { timeout: 5000 }).click();
 
-        cy.get('a[data-cy="project-description-view-link"]', { timeout: 5000 }).click();
-
-        // Wait for the edit button to be visible and click it
-        cy.get('#project-summary-submit').then(($button) => {
-            $button[0].click();
-        });
+    // Wait for the edit button to be visible and click it
+    cy.get("#project-summary-submit").then(($button) => {
```

Comment:
I did not mean to edit this file. I think this is just eslint fixing formatting from when I did a merge.